src/transformState.js

```diff
@@ -5,7 +5,24 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    switch (true) {
```

Comment:
This type of check is valid, but we can use switch in more convenient way, just pass action.type here